http-gateways/PATH_GATEWAY.md

```diff
@@ -388,6 +392,10 @@ Returned directive depends on requested content path and format:
   - If TTL value is unknown, implementations are free to set it to a static
     value, but it should not be lower than 60 seconds.
 
+- `Cache-Control: no-cache, no-transform`  should be returned with
+  `application/x-tar` responses if the TAR response, is not guaranteed
```

Comment:
nit: grammar, should be no comma after TAR response. Change to `responses if the TAR response is not guaranteed to be deterministic.`